MIGRATION.md

```diff
@@ -1,5 +1,90 @@
 # Migration Guide
 
+## Version `6.*` to Version `7.*`
```

Comment:
Could you change this to `Version 2 to Version 3` so that its consistent with the existing migration guide?

Reply:
done